src/App.tsx

```diff
@@ -1,16 +1,60 @@
 import React from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [query, setQuery] = React.useState('');
+  const [isDropedDown, setIsDropedDown] = React.useState(false);
+  const [selectedPerson, setSelectedPerson] = React.useState<Person | null>(
```

Comment:
```suggestion
  const [query, setQuery] = useState('');
  const [isDropedDown, setIsDropedDown] = useState(false);
  const [selectedPerson, setSelectedPerson] = useState<Person | null>(
```

Reply:
ty, but this is a matter of taste, not rationality.

Reply:
But would be better stick to react doc 
![Uploading image.png…]()